src/app/shared/components/template/components/drawer/drawer.component.ts

```diff
@@ -7,22 +7,20 @@ import { TemplateBaseComponent } from "../base";
   templateUrl: "./drawer.component.html",
   styleUrls: ["./drawer.component.scss"],
 })
-export class TmplDrawerComponent extends TemplateBaseComponent implements OnInit, AfterViewInit {
-  @ViewChild("drawer", { read: ElementRef }) drawer: ElementRef;
+export class TmplDrawerComponent extends TemplateBaseComponent implements OnInit {
+  @ViewChild("drawer", { read: ElementRef, static: false }) drawer: ElementRef;
 
   isOpen = false;
-  openHeight = 0;
   style: string;
 
-  ngOnInit() {
-    this.getParams();
+  get openHeight() {
```

Comment:
nit(non-blocking)
I don't see this accessed from the component html so should it be marked as private? also does it need a getter or would it work fine as a regular method? Generally getters are more associated with lookups or alisasing to quickly access things from the frontend component, so whilst the code still works fine it does make me think that openHeight exists in the template somewhere.